### PR TITLE
bugzilla plugin: list validations run on refresh

### PR DESCRIPTION
**Pull request by AlexNPavel**
This PR makes the bugzilla plugin more verbose on successful validation, listing which validations it ran to determine that the bug was valid.

/cc @stevekuznetsov

### CODE REVIEW

File: prow/plugins/bugzilla/bugzilla.go [hunk 1]
```diff
@@ -546,6 +557,12 @@ func validateBug(bug bugzilla.Bug, dependents []bugzilla.Bug, options plugins.Bu
 			was = "is"
 		}
 		errors = append(errors, fmt.Sprintf("expected the bug to %sbe open, but it %s", not, was))
+	} else if options.IsOpen != nil {
+		was := "isn't"
+		if *options.IsOpen {
+			was = "is"
+		}
+		validations = append(validations, fmt.Sprintf("bug %s open", was))
```
Comment (stevekuznetsov) [inline prow/plugins/bugzilla/bugzilla.go]: This is missing the expected state

File: prow/plugins/bugzilla/bugzilla_test.go [hunk 2]
```diff
@@ -802,6 +814,10 @@ Instructions for interacting with me using PR comments are available [here](http
 			expectedLabels: []string{"bugzilla/valid-bug"},
 			expectedComment: `org/repo#1:@user: This pull request references [Bugzilla bug 123](www.bugzilla/show_bug.cgi?id=123), which is valid.
 
+Validations run on this bug:
+	- dependent bug [Bugzilla bug 124](www.bugzilla/show_bug.cgi?id=124) is in the state VERIFIED, which is one of the valid states (VERIFIED)
+	- bug has dependents
```
Comment (stevekuznetsov) [inline prow/plugins/bugzilla/bugzilla_test.go]: This is only an implicit requirement, right?

File: prow/plugins/bugzilla/bugzilla_test.go [hunk 3]
```diff
@@ -626,6 +626,8 @@ Instructions for interacting with me using PR comments are available [here](http
 			expectedLabels: []string{"bugzilla/valid-bug"},
 			expectedComment: `org/repo#1:@user: This pull request references [Bugzilla bug 123](www.bugzilla/show_bug.cgi?id=123), which is valid.
 
+No validations were run on this bug.
```
Comment (stevekuznetsov) [inline prow/plugins/bugzilla/bugzilla_test.go]: I would expect at least one complicated test case in this which has a long list.

File: prow/plugins/bugzilla/bugzilla_test.go (hunk 2 above)
Comment (AlexNPavel) [reply prow/plugins/bugzilla/bugzilla_test.go]: Yes, but it's a validation we run, so I added it to the list. We can remove it if you think it's unnecessary.

**Comment by petr-muller** [comment]
LOL what, I'm an approver?

**Pull request merged**